yarn.lock

```diff
@@ -1500,14 +1500,14 @@
     "@volar/typescript" "~1.10.0"
     "@vue/language-core" "1.8.8"
 
```

Comment:
Why didn't the package.json change?

Reply:
The original PR only affected lockfile: https://github.com/Kong/kongponents/pull/1619/files